acj/static/modules/answer/answer-create-modal-partial.html

```diff
@@ -0,0 +1,9 @@
+<modal-cancel-button modal-instance="modalInstance" ng-show="modalInstance !== undefined"></modal-cancel-button>
```

Comment:
Looks like answer create and edit partial have only one difference. Could you use one and pass the text from the controller to reduce the duplicate code?